recipes/orjson/build.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -ex
+
+# Fun times -- by default, Rust/Cargo tries to link executables on Linux by
+# invoking `cc`. An executable of this name is not necessarily available. By
+# setting a magic environment variable, we can override this default.
```

Comment:
Let's replace these comments with a link to the rust feedstock.

Reply:
Done